CHANGELOG.md

```diff
@@ -2,6 +2,19 @@
 
 All versions are tagged by the major Postgres version, plus an individual semver for this library itself.
 
+## 15-4.2.4   2023-12-20
+
+* Add Token `ASCII_36` [#211](https://github.com/pganalyze/libpg_query/pull/211), [#219](https://github.com/pganalyze/libpg_query/pull/219)
+* Fix failing type cast in WebAssembly builds [#223](https://github.com/pganalyze/libpg_query/pull/223)
```

Comment:
```suggestion
* Normalize: Fix incorrect type cast [#223](https://github.com/pganalyze/libpg_query/pull/223)
```

(I don't think its a WebAssembly specific issue, just that it surfaced there -- its not clear why this worked otherwise)